src/test/resources/org/rust/ide/formatter/fixtures/indent_where_clause_off.rs

```diff
@@ -0,0 +1,4 @@
+impl moo {
+    pub fn do_stuff<T>(&self, item: &T)
+where T: ToString {}
+}
```

Comment:
Looks like we can simplify test examples and drop `impl` blocks

Reply:
I've done what you asked. The test above mine had the impl block so I did the same. It adds some visual clarity to the expected indentation, but it isn't strictly necessary.